### ISSUE TITLE
ELI5 me the next central bank decision to cut reward

### ISSUE THREAD

**Issue by sammy007** [sammy007/open-ethereum-pool#386]
I have no time for this mess, so please explain me upcoming changes so I can patch this pool in a free time. need a lt'dr and a link with a proof.

**Comment by mikeyb** [comment]
https://eips.ethereum.org/EIPS/eip-1234

Reward being dropped to 2ETH, this will delay difficulty rise for another 12 months so they can kick the can down the road

You can see all the proposals here: https://eips.ethereum.org/all

This is under the Accepted section.

**Comment by sammy007** [comment]
Did the decide on what block height already?

**Comment by mikeyb** [comment]
As far as I know `CNSTNTNPL_FORK_BLKNUM` has not been set in code yet.  Will update when I see this change come thru if you don't see it first

**Comment by agentnotneeded** [comment]
@sammy007 What would you charge to set this up for me?

**Comment by 9600-** [comment]
Didn’t we go through this on the Byzantium fork? 
Its a pretty easy matter of changing the block reward as outlined in the issues associated with that fork. Should be trivial matter of adding a class for CNSTNTNPL to UNLOCKER.go and making sure the reward is set to 3000000000000000000. 

That’s pretty much what we did for Byzantium (5 to 3) and we’ve been minting blocks without issue. 

Here’s the beginning of the talk back on the Byzantium fork:
https://github.com/sammy007/open-ethereum-pool/issues/174

**Comment by mikeyb** [comment]
Yes.  He was asking for official information, like what block is the change happening on, and what will the necessary changes be.

**Comment by ghost** [comment]
let's clear up: nothing to putch here? We just change the reward from 3..... to 2....

**Comment by sammy007** [comment]
```go
const constantinopleHardForkHeight = ???
var constantinopleReward = math.MustParseBig256("2000000000000000000")

func getConstReward(height int64) *big.Int {
	if height >= constantinopleHardForkHeight {
		return new(big.Int).Set(constantinopleReward)
	}
	if height >= byzantiumHardForkHeight {
		return new(big.Int).Set(byzantiumReward)
	}
	return new(big.Int).Set(homesteadReward)
}
```

**Comment by ghost** [comment]
Didn't know that you've opened a issue for this 😂 , yes I think EIP-1234 is too centralized this is the perfect situation for ethereum now

![e48fb7bdb7476a9c16b2546f8bc2e31b](https://user-images.githubusercontent.com/39383087/45598660-99f9c880-ba1a-11e8-990a-f45ec77af4e1.gif)

**Comment by sammy007** [comment]
Added new block reward for Constantinople monetary policy scam. No clue if there is other updates needed for rpc or something else.
